stage1/init/init.go

```diff
@@ -648,7 +660,26 @@ func stage1(rp *stage1commontypes.RuntimePod) int {
 	}
 	diag.Printf("canMachinedRegister %t", canMachinedRegister)
 
-	args, env, err := getArgsEnv(p, flavor, canMachinedRegister, debug, n)
+	// --ipc=[auto|private|parent]
+	// default to private
+	parentIPC := false
+	switch p.IPCMode {
+	case "parent":
+		parentIPC = true
+	case "private":
+		parentIPC = false
+	case "auto":
+		fallthrough
+	case "":
+		parentIPC = false
+	default:
+		log.Fatalf("unknown value for --ipc parameter: %v", p.IPCMode)
+	}
+	if parentIPC && flavor == "kvm" {
```

Comment:
Should we do this for the "fly" flavor? That is, if we pass `--ipc=private`

Right now we just display a warning.

Reply:
Also, we won't get here because stage0 will prevent us from passing `--ipc=` before, maybe it's good to keep it for extra safety?

Reply:
@iaguis "fly" is implemented in the `stage1_fly` directory and it does not link with `stage1/init/init.go`.

Reply:
Yes, what I mean is if we should do something similar in `stage1_fly` (that is, explicitly refuse `ipc=private`)

Reply:
oh... could it be left for the follow-up issue https://github.com/rkt/rkt/issues/3789 ?

Reply:
sounds good!